recipes/mdi/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "mdi" %}
+{% set version = "1.4.34" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MolSSI/MDI_Library/archive/v{{ version }}.tar.gz
+  sha256: 0128a9f2577f6da7f154b10a3410e9906e062976896cfb5bb7c60bf28e7b3309
+
+build:
+  number: {{ build }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib("c") }}
+    - cmake >=3.13
+    - make
+    - ninja
+    - python {{ python }}
+    - {{ mpi }}
+  host:
+    - python {{ python }}
+    - {{ mpi }}
+  run:
+    - python {{ python }}
```

Comment:
Since this has a run time dependency on Python, does this mean it provides Python bindings?  If so, can we do an `import` test to verify they work?  That would be a little more substantive than testing for the existence of files.

Reply:
Yes, the package has Python bindings.  I've added the import test.